internal/pkg/testutil/testing.go

```diff
@@ -9,26 +9,31 @@ import (
 	"github.com/segmentio/terraform-docs/internal/pkg/tfconf"
 )
 
-// GetExpected returns 'example' Module and expected Golden file content
-func GetExpected(goldenFile string) (*tfconf.Module, string, error) {
-	path, err := getExampleFolder()
+// GetModule returns what is generated by the test params
+func GetModule(options *tfconf.Options) (*tfconf.Module, error) {
+	path, err := GetExampleFolder()
 	if err != nil {
-		return nil, "", err
-	}
-	options := &tfconf.Options{
-		Path: path,
+		return nil, err
 	}
+	options.Path = path
+	options.OutputValuesPath = path + options.OutputValuesPath
+
 	module, err := tfconf.CreateModule(options)
 	if err != nil {
-		return nil, "", err
+		return nil, err
 	}
+	return module, err
+}
 
+// GetExpected returns 'example' Module and expected Golden file content
+func GetExpected(goldenFile string) (string, error) {
 	expected, err := readGoldenFile(goldenFile)
```

Comment:
Also having a separate `readGoldenFile` seems to be irrelevant and its content can be moved here. something like this:

```go
// GetExpected returns 'example' Module and expected Golden file content
func GetExpected(format, name string) (string, error) {
	path := filepath.Join(testDataPath(), format, name+".golden")
	bytes, err := ioutil.ReadFile(path)
	if err != nil {
		return "", err
	}
	return string(bytes), nil
}
```
and delete `readGoldenFile` function altogether.

Reply:
Was wondering what the `format` var is meant to be? Previously, `GetExpected` was taking the golden file name less the extension as the arg.

Reply:
That was copy paste issue :see_no_evil: which we will need in the PR #195 which refactors stuff. For this PR only `name` suffices.

Reply:
:smile:  sounds good! I figured, but wanted to clarify. Will be pushing shortly!